mucommander-protocol-gcs/src/test/java/com/mucommander/commons/file/protocol/gcs/GoogleCloudStorageBucketTest.java

```diff
@@ -0,0 +1,56 @@
+package com.mucommander.commons.file.protocol.gcs;
+
+import com.google.cloud.storage.BlobInfo;
+import com.mucommander.commons.file.FileURL;
+import org.junit.Test;
+
+import java.net.MalformedURLException;
+
+import static org.testng.Assert.assertEquals;
+
+public class GoogleCloudStorageBucketTest {
+
+    @Test
+    public void getBucketName() throws MalformedURLException {
+        var file1 = FileURL.getFileURL("gcs://project-id/bucket-name1/folder/folder2/file.txt");
+        var file2 = FileURL.getFileURL("gcs://project-id/bucket-name2/folder/folder2");
+        var file3 = FileURL.getFileURL("gcs://project-id/bucket-name3/");
+        var file4 = FileURL.getFileURL("gcs://project-id/bucket-name4");
+        var file5 = FileURL.getFileURL("gcs:///bucket-name5");
```

Comment:
is it a valid path?

Reply:
It is not, but the thought is that project-id shouldn't affect the bucket's name, even if missing.